modules/admin_manual/pages/configuration/server/occ_command/background_jobs_selector.adoc

```diff
@@ -11,18 +11,19 @@ background
  background:webcron    Use webcron to run background jobs
 ----
 
-This example selects Ajax:
+== Example
```

Comment:
Shouldnt it be plural as you have more than one example?

Reply:
Right you are. Will update it.

Reply:
Done!